cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -485,6 +488,20 @@ def create(self, catalog_item_class, *args, **kwargs):
         """
         cat_item = self.instantiate(catalog_item_class, *args, **kwargs)
         view = navigate_to(cat_item, 'Add')
+        if field_entry_point:
+            if provider_type == 'cloud':
+                view.basic_info.field_entry_point.fill("")
+                view.basic_info.modal.tree.click_path('Datastore', 'ManageIQ (Locked)', 'Cloud',
+                                                      'VM', 'Provisioning', 'StateMachines',
+                                                      'ProvisionRequestApproval', 'Default')
+            elif provider_type == 'infra':
+                view.basic_info.field_entry_point.fill("")
+                view.basic_info.modal.tree.click_path('Datastore', 'ManageIQ (Locked)',
+                                                      'Infrastructure', 'VM', 'Provisioning',
```

Comment:
Required:
In this tree path `Cloud/Infrastructure` changes. So, better to manipulate that only... what you say?

Reply:
Yes, this is very useful. I am updating PR with this suggestion.